src/index.ts

```diff
@@ -0,0 +1,104 @@
+import express from 'express';
+import { readdir } from 'fs/promises'
+import path from 'path';
+import helmet from 'helmet';
+import { cwd } from 'process';
+import { extname, join } from 'upath'
+
+import 'dotenv/config'
+
+
+let app = express();
+const PORT = 3000;
+
+app.use(helmet())
+app.use(express.json())
+
+
```

Comment:
we can set the root directory here, 
```suggestion
const rootDirectory = process.env.ROOT_DIR ?? process.cwd()
```